.github/workflows/docker-tag-v1.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Docker tag
+
+on:
+  workflow_call:
+    inputs:
+      aws_region:
+        type: string
+        default: "eu-central-1"
+      ci_iam_role:
+        type: string
+        default: ""
+      docker_image_name:
+        type: string
+        required: true
+      docker_tag_from:
+        type: string
+        required: true
+      docker_tag_to:
+        type: string
+        required: true
```

Comment:
Do we need to prefix these input variables name with `docker`? This looks redundant to me as we're in the context of a `Docker tag` action here :-)

Reply:
No we don't, we just need to be consistent

Reply:
https://github.com/sencrop/github-workflows/pull/89